lib/hooks/queries/constants.ts

```diff
@@ -1,8 +1,11 @@
 import { MarketWhereInput } from "@zeitgeistpm/indexer";
+import { isWSX } from "lib/constants";
 
 export const marketMetaFilter: MarketWhereInput = {
   question_isNull: false,
   question_not_eq: "",
   categories_isNull: false,
   hasValidMetaCategories_eq: true,
+  baseAsset_eq: isWSX ? '{"foreignAsset":3}' : undefined,
```

Comment:
Could you pull this: "{"foreignAsset":3}" out to it's own variable? It probably won't be that on production